project_workflow_default_state_per_group/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2017 - 2018 Modoolar <info@modoolar.com>
+# License LGPLv3.0 or later (https://www.gnu.org/licenses/lgpl-3.0.en.html).
+{
+    "name": "Project Workflow Default Stage Per Security Group",
+    "summary": "This module enables default task states per security group.",
+    "category": "Project",
+    "version": "11.0.1.0.0",
+    "license": "LGPL-3",
+    "author": "Odoo Community Association (OCA), Modoolar",
+    "website": "https://www.modoolar.com/",
```

Comment:
@pnajman-modoolar IMHO Change URL as per [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L10)